Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in flex_commerce_api-rails.gemspec
+# Specify your gem's dependencies in shift_commerce-rails.gemspec
+gem "flex_commerce_api", git: "https://github.com/flex-commerce/flex-ruby-gem.git", tag: 'v0.3.22'
```

Comment:
This dependency should be added to the gemspec not the gemfile.  The gemfile is for development use - it doesnt get used when using the gem.
Also, the gemspec does not accept github urls just to add to the complication, which is one of the reasons why the flex_commerce_api gem is not included yet. When it has a proper entry in rubygems this wont be a problem